app/services/offender_service.rb

```diff
@@ -93,36 +36,15 @@ def self.get_multiple_offenders(offender_ids)
     }
   end
 
-  def self.get_multiple_offenders_as_hash(offender_ids)
```

Comment:
Is there a reason this is removed? It's not related to a Prison, assume it's because it doesn't do much?

Reply:
it was completely unused